app/src/main/java/com/greenmonkeys/handmade/payout_denominations/PesoWallet.java

```diff
@@ -0,0 +1,88 @@
+package com.greenmonkeys.handmade.payout_denominations;
+
+public class PesoWallet {
+
+    private int fiveHundred,twoHundred, oneHundred, fifty, twenty, ten, five, two, one, fiftyCent, twentyCent, tenCent, fiveCent, oneCent;
+
+    public PesoWallet(int fiveHundred, int twoHundred, int oneHundred,
```

Comment:
PesoWallet constructor should take one (or two) arrays/maps of values. If you NEED to parse it back out into individual variables (which you shouldn't), do so in the constructor method of PesoWallet.